### PR TITLE
Improve logging 

### PR DESCRIPTION
**Pull request by SarahOuologuem**
Hi! I added some logging to the spatial workflows and wanted to check with you @bio-la whether this is what you're thinking of before I continue with the other workflows.

### CODE REVIEW

**Review by bio-la — COMMENTED** [review]
these look good checkpoint messages to me!

File: panpipes/python_scripts/batch_correct_bbknn.py [hunk 1]
```diff
@@ -61,47 +60,54 @@
...
+        L.info("Setting n PCS to %i" % int(n_pcs))   
+    L.info("Scaling data") 
     sc.pp.scale(adata)
+    L.info("Computing PCA")
     sc.tl.pca(adata, n_comps=n_pcs, 
                     svd_solver='arpack', 
                     random_state=0) 
 
 
-L.info("preparing for integration")
```
Comment (bio-la) [inline panpipes/python_scripts/batch_correct_bbknn.py:82]: put this line back please

File: panpipes/python_scripts/batch_correct_harmony.py [hunk 2]
```diff
@@ -56,14 +56,12 @@
 
 args, opt = parser.parse_known_args()
 
-
-L.info("reading data and starting integration pipeline with script: ")
-L.info(os.path.basename(__file__))
-L.info("Running with options: %s", args)
+L.info(args)
```
Comment (bio-la) [inline panpipes/python_scripts/batch_correct_harmony.py:59]: I prefer the vesion of this command with a preamble "running with params"

**Review by bio-la — CHANGES_REQUESTED** [review]

File: panpipes/python_scripts/batch_correct_bbknn.py [hunk 3]
```diff
@@ -44,15 +44,14 @@
 args, opt = parser.parse_known_args()
 
 
+L.info(args)
 
-L.info("reading data and starting integration pipeline with script: ")
-L.info(os.path.basename(__file__))
-L.info("Running with options: %s", args)
```
[the remark this answers is not in the document]
Comment (SarahOuologuem) [reply panpipes/python_scripts/batch_correct_bbknn.py:50]: ok! would you also add the script to the log, i.e.` L.info(os.path.basename(__file__))`?

File: panpipes/panpipes/pipeline_clustering.py [hunk 4]
```diff
@@ -260,7 +259,7 @@ def gen_marker_jobs(infile, outfile, base_dir):
 @follows(collate_mdata)
 @transform(gen_marker_jobs,
            regex("(.*)/(.*)/(.*)_markers.txt"),
-           r"logs/\1_\2_\3_markers.log",
+           r"logs/8_find_markers_\1_\2_\3.log",
```
[the remark this answers is not in the document]
Comment (SarahOuologuem) [reply panpipes/panpipes/pipeline_clustering.py:284]: it looks like this
clustering_logs

File: panpipes/python_scripts/batch_correct_bbknn.py (hunk 1 above)
Comment (SarahOuologuem) [reply panpipes/python_scripts/batch_correct_bbknn.py:82]: done

**Comment by SarahOuologuem** [comment]
@bio-la  I now added 
`L.info("Running with params: %s", args)` to every file instead of `L.info(args)`

**Review by bio-la — APPROVED** [review]

**Comment by bio-la** [comment]
@SarahOuologuem awesome job!

**Pull request merged**